OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Validations.Add.cs

```diff
@@ -140,6 +140,46 @@ public async void ShouldThrowValidationExceptionOnCreateWhenTeacherIsNullAndLogI
             this.storageBrokerMock.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnAddwhenTeacherStatusIsInvalidAndLogItAsync()
+        {
+            //given
+            DateTimeOffset date = GetRandomDateTime();
+            Teacher randomTeacher =  CreateRandomTeacher(date);
+            var invalidTeacher = randomTeacher;
+            invalidTeacher.Status = GetInValidTeacherStatus();
+            var invalidTeacherException = new InvalidTeacherException();
+                
+                invalidTeacherException.AddData(
+                    key:nameof(Teacher.Status),
+                    values: "Value is invalid");
+
+            var expectedTeacherValidationException = new TeacherValidationException(invalidTeacherException);
```

Comment:
Push this guy to the next line with one tab

Reply:
It is fixed